airflow/providers/apache/kafka/sensors/kafka_sensor.py

```diff
@@ -0,0 +1,76 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from cached_property import cached_property
+
+from airflow.providers.apache.kafka.hooks.kafka_consumer_hook import KafkaConsumerHook
+from airflow.sensors.base_sensor_operator import BaseSensorOperator
+from airflow.utils.decorators import apply_defaults
+
+
+class KafkaSensor(BaseSensorOperator):
+    """
+    Consumes the Kafka message with the specific topic
+    """
+    DEFAULT_HOST = 'kafka1'
+    DEFAULT_PORT = 9092
+    templated_fields = ('topic',
+                        'host',
+                        'port',
+                        )
```

Comment:
```suggestion
    templated_fields = ('topic', 'host', 'port', )
```

They can fit in one line :)